example/src/app.js

```diff
@@ -66,6 +76,9 @@ class App extends React.Component {
           onSuggestNoResults={this.onSuggestNoResults}
           location={new google.maps.LatLng(53.558572, 9.9278215)}
           radius="20" />
+        <div>
+          {this.state.selectedSuggest && this.state.selectedSuggest.label}
+        </div>
```

Comment:
Please leave this out of the example. It helps to demonstrate the issue but is not for showcasing the component.